components/milestone/Milestone.js

```diff
@@ -12,17 +14,11 @@ const useStyles = makeStyles(theme => ({
 export default function Milestone(props) {
   const classes = useStyles();
   const { milestone } = props;
+  const { category, title, goal } = milestone;
```

Comment:
You can also grab the `slug` from the milestone. This is what you will provide to the `ActivityHeader` for the `milestoneType` prop